src/shim/shim-config.ts

```diff
@@ -245,47 +210,6 @@ class Config implements IConfig {
                 process.env["APPLICATION_INSIGHTS_NO_STANDARD_METRICS"] = "disable";
             }
         }
-        // NATIVE METRICS
-        if (typeof (this.enableAutoCollectExtendedMetrics) === "boolean") {
```

Comment:
If you are removing native metrics please remove any related code, including all types, only updating the config is not enough